.github/workflows/cron-update-rust.yml

```
@@ -0,0 +1,41 @@
+name: Update rust version
+on:
+  schedule:
+    - cron: "0 0 15 * *" # At 00:00 on day-of-month 15.
+  workflow_dispatch: # allows manual triggering
+jobs:
+  format:
+    name: Update rustc
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - name: Update rust.yml to use latest stable
```

Should this be `rust-version` instead of `rust.yml`, on line 39 also? 
```suggestion
      - name: Update rust-version to use latest stable
```

---

Yes, thanks. On line 39 I changed it to refer to the actual workflow file `cont_integration.yml`